vignettes/announcing-tidycdisc-0-2-1.Rmd

```diff
@@ -0,0 +1,189 @@
+
+Below is an example when `ATPT` exists in the uploaded `ADVS` data. When applicable, you can select "NONE" to revert to the old behavior of aggregating all time points, i.e. avoiding the breakdown by time point. Conversely, you can select one specific time point to display that measurement at each visit. Below, we select the "After standing for 1 minute" time point and displaying "All" visits in our `MEAN` stat block:
+ 
+<br> 
+
+```{r, echo=FALSE, out.width="80%", fig.align = 'center'}
+knitr::include_graphics("figures/blog-0-2-1/atpt_dropdown.png")
+```
+
+<br>
+
+```{r, echo=FALSE, out.width="90%", fig.align = 'center'}
+knitr::include_graphics("figures/blog-0-2-1/atpt_output.png")
+```
+
+<br>
+
+Last, selecting “ALL” will display output for each time point value available. In the example below example, that would produce a table containing a descriptive statistics on diastolic blood pressure while (1) lying down for 5 minutes, (2) after standing for 1 minute, and again (3) after standing for 3 minutes. Notice how these three outputs are created for each visit:
+
+<br> 
+
+```{r, echo=FALSE, out.width="80%", fig.align = 'center'}
+knitr::include_graphics("figures/blog-0-2-1/select_all.png")
+```
+
+<br>
+
+```{r, echo=FALSE, out.width="90%", fig.align = 'center'}
+knitr::include_graphics("figures/blog-0-2-1/select_all_output.png")
+```
+
+<br>
+
+### Simplified function call to reproduce tables
+
+If you're using `tidyCDISC` as an R script generator for reproducing your favorite tables, then your R files just a little cleaner. We've engineered a user-friendly function called `tidyCDISC::tg_gt()` that consolidates a lot of the code used to easily generate a table from the "block data". If you don't know what block data is, it's basically just configuration data.frame you've developed via the drag and drop interface in the app. In other words, "Block data" tells `tidyCDISC` how to build the table, but don't worry about it too much for now. The next release of `tidyCDISC` will include code that helps users manually create "block data" in a lot more intuitive way.
+
+For more information on this releases' new function, run `?tidyCDISC::tg_gt` in your R console AND download an R script from the app to see it in action.
+
+
+### Other enhancements 
+
+- Added `OS Health` and `riskmetric` badges to the `README`
+
+- Added an app.R file so that the Shiny app can be run directly from `shiny::runGitHub()` or `runUrl()`
+
+- Populate Table title automatically when a "standard analysis" is selected
+
+- “ALL” is now the default visit selection for standard analysis 41
+
+- Fixed bug when filtering module in Population Explorer where columns for non-selected datasets would display erroneously
+
+- Fixed bug where erroneously variable was listed as a grouping option in the Population Explorer
+
+- Fixed bug accessing files in "app/www/"
+
+- Fixed bug where draggable blocks were not working without inclusion of datasets with ATPT
```

Comment:
I'm fine with including this note, but the bug was only present when the ATPT dropdown was added to the application. This bug would not have been present in the previous CRAN release.

Reply:
That's true. I think it was available on the previous `master` version (not released on CRAN). I'm fine with removing it.